BICCN-policies/dataset-metadata.yaml

```diff
@@ -0,0 +1,85 @@
+identifier: REQUIRED ## Post upload (or during dandi organize)
+name: REQUIRED
+description: REQUIRED
+contributors: # required for author and contact
+- orcid: # REQUIRED
+  roles: # REQUIRED from https://casrai.org/credit/ + maintainer, contact
```

Comment:
Thanks for the link to casrai.org/credit - looks usefule!  Would you just want to use the role alone, or include optional additional text, much as is provided in an acknowledgement in a paper?